generate-training-data.py

```diff
@@ -1,13 +1,27 @@
 import os
 import cv2
 
-# write your code here to open your laptop's camera
-# and store images for rock paper scissors
-
-# I would highly recommend storing these images in folders stucture like below:
-# training_data\
-#           |-- empty\
-#           |-- rock\
-#           |-- paper\
-#           |-- scissors\
-# having a folder structure like this will make it easy for you to read the images while pre-processing
+cap = cv2.VideoCapture(0)
+count = 0
+while (cap.isOpened()):
+    rep, frame = cap.read()
+
+    if rep == True:
+
+        #gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+        #out.write(frame)
+        font = cv2.FONT_HERSHEY_PLAIN
+        frame = cv2.rectangle(frame, (100,100), (400,400),(255,0,0), 5)
+        frame = cv2.putText(frame, "count="+str(count/5), (300, 420), font,1 , (255, 0, 0), 3)
+        if count%5 == 0:
```

Comment:
I see that you are saving every 5th image that's not a bad idea, but your program keeps running unless you hit `q` try giving a way for it to stop after a certain number of images have been collected.